backend/inspirehep/records/marshmallow/literature/utils.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from inspirehep.records.api import InspireRecord
+
+
+def get_parent_record(data):
+    if data.get("doc_type") == "inproceedings":
+        conference_records = InspireRecord.get_linked_records_from_dict_field(
+            data, "publication_info.conference_record"
+        )
+        conference_record = next(conference_records, {})
+        return conference_record
+
+    book_records = InspireRecord.get_linked_records_from_dict_field(
+        data, "publication_info.parent_record"
+    )
+    book_record = next(book_records, {})
```

Comment:
is this same as `book_records or book_records[0]` ?

Reply:
As I only moved this code from bibtex serializer I would preffer not to discuss this code but mostly it's the same, except here if book_records is empty it returns empty dict.